### PR TITLE
CONCF-1063 | Style section template

### PR DESCRIPTION
**Pull request by vforge**
needs to wait for 1062

### CODE REVIEW

**Comment by bognix** [comment]
LGTM

**Comment by Warkot** [comment]
LGTM :water_buffalo:

File: front/styles/main/component/_curated-content-editor.scss [hunk 1]
```diff
@@ -89,61 +92,81 @@ $search-results-load-more-button-width: 100vw - 2 * $search-results-horizontal-p
...
 		}
 	}
 
-	.section-desc {
-		display: flex;
-		flex: 1;
-		flex-direction: column;
-		height: $image-size;
-		justify-content: space-between;
+	&:first-of-type .arrow-wrapper:first-of-type,
```
Comment (hakubo) [inline front/styles/main/component/_curated-content-editor.scss]: nice

**Comment by hakubo** [comment]
LGTM

**Pull request merged**